products/healthcare/api.yaml

```diff
@@ -540,3 +540,74 @@ objects:
         'Creating a HL7v2 Store':
           'https://cloud.google.com/healthcare/docs/how-tos/hl7v2'
       api: 'https://cloud.google.com/healthcare/docs/reference/rest/v1/projects.locations.datasets.hl7V2Stores'
+  - !ruby/object:Api::Resource
+    name: 'ConsentStore'
+    kind: "healthcare#consentStore"
+    base_url: "{{dataset}}/consentStores?consentStoreId={{name}}"
+    self_link: "{{dataset}}/consentStores/{{name}}"
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: |
+      The Consent Management API is a tool for tracking user consents and the documentation associated with the consents. 
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      method_name_separator: ':'
+      fetch_iam_policy_verb: :GET
+      parent_resource_attribute: 'consent_store_id'
+      import_format: ["{{dataset}}/consentStores/{{name}}", "{{name}}"]
```

Comment:
Due to some arcane ways that resource generation and import/url formats works if you change `{{dataset}}` in these three fields to `{{%dataset}}` the tests will pass.

Adding the `%` basically tells the code that the field can contain `/` characters, which lets the import/URL qualification work with `dataset` which is a long-form name